### ISSUE TITLE
azurerm_kusto_cluster missing parameter allowedIpRangeList

### ISSUE THREAD

**Issue by liranms** [hashicorp/terraform-provider-azurerm#16420]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

There's a security property on kusto cluster which is very useful, called allowedIpRangeList, it is a list of ip addresses or CIDR that are allowed access to the cluster when the cluster has internet access disabled
on az cli it looks like this:
```    --allowed-ip-range-list                                       : The list of ips in the format of
                                                                    CIDR allowed to connect to the
                                                                    cluster.
```

### New or Affected Resource(s)/Data Source(s)

azurerm_kusto_cluster

### Potential Terraform Configuration

```hcl
allowed_ip_range_list = ["123.123.123.123/24", "1.2.3.4"]
```

**Comment by ms-henglu** [comment]
Hi @liranms ,

This feature has been supported in https://github.com/hashicorp/terraform-provider-azurerm/blob/main/CHANGELOG.md#3150-july-21-2022, would you please give it a try? Thanks!

**Comment by rcskosir** [comment]
@liranms Thank you for opening this issue. It looks like this has been resolved as of [v3.15.0](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/CHANGELOG.md#3150-july-21-2022). As such, I am going to mark this issue as closed.
